framework/test/randoop_coverage.sh

```diff
@@ -65,6 +72,13 @@ suite_num=1
 #rm -f $master_coverage
 
 for pid in "${projects[@]}"; do
+    if (( all_bids == 1 )); then
+        bids=($(defects4j query -p $pid))
```

Comment:
Using the `defects4j bids` command seems more robust, in case the default for `query` changes in the future.